server/games/ladder_game.py

```diff
@@ -34,3 +36,18 @@ def get_army_score(self, army: int) -> int:
         as 1 for win and 0 for anything else.
         """
         return self._results.victory_only_score(army)
+
+    def _outcome_override_hook(self) -> Optional[List[GameOutcome]]:
+        if not config.LADDER_1V1_OUTCOME_OVERRIDE or len(self.players) > 2:
```

Comment:
Checking the player count isn't actually necessary since `max_players` is set to two, but with tmm coming up I wanted to be sure, if it adapts this class.